pom.xml

```diff
@@ -252,7 +252,7 @@
                 <artifactId>tomcat7-maven-plugin</artifactId>
                 <version>2.2</version>
                 <configuration>
-                    <port>8081</port>
+                    <port>8082</port>
```

Comment:
This change would cause everybody else to switch their port number. This should not be committed. Can you please change the phone number and push again ?